source/API_Reference/Web_API_v3/How_To_Use_The_Web_API_v3/authentication.md

```diff
@@ -31,3 +32,39 @@ Authorization: Bearer Your.API.Key-HERE
 {% codeblock lang:bash %}
 curl -X "GET" "https://api.sendgrid.com/v3/templates" -H "Authorization: Bearer Your.API.Key-HERE" -H "Content-Type: application/json"
 {% endcodeblock %}
+
+{% anchor h3 %}
+On-Behalf of Subuser
+{% endanchor %}
+
+The on-behalf-of header allows you to make calls for a particular subuser through the parent account, this can be useful for automating bulk updates, or administering a subuser without changing authentication in your code. In the header you are passing, you will simply need to add: `On-Behalf-Of: subuser_username`
```

Comment:
account**;** this can be useful for automating bulk updates~~,~~ or administering...
(upgrade comma after `account` to a semicolon and get rid of comma after `updates`)